test/e2e/networking.go

```diff
@@ -216,6 +216,8 @@ var _ = framework.KubeDescribe("Networking", func() {
 	// Marked with [Flaky] until the tests prove themselves stable.
 	framework.KubeDescribe("[Flaky] Granular Checks", func() {
 
+		connectivityTimeout := 10
```

Comment:
I've chosen 10s because that's what we've been using in the originating openshift test [1] and haven't had problems with flakes.  The connectivity check only starts after the pod has been verified to be up by LaunchWebserverPod and a single http call should be sub-second unless there are severe connectivity problems between the host running the tests and the target cluster.

1: https://github.com/openshift/origin/blob/master/test/extended/networking/sanity.go#L11